Kitodo/src/main/java/org/kitodo/production/forms/MigrationForm.java

```diff
@@ -200,20 +202,27 @@ public String convertTasksToWorkflow(String tasks) {
                 PrimeFaces.current().executeScript("PF('confirmWorkflowPopup').show();");
                 return this.stayOnCurrentPage;
             }
-        } catch (DAOException e) {
+        } catch (DAOException | IOException | WorkflowException e) {
             Helper.setErrorMessage(ERROR_READING, new Object[] {ObjectType.TEMPLATE.getTranslationSingular() }, logger,
                 e);
             return this.stayOnCurrentPage;
         }
         return createNewWorkflow();
     }
 
-    private boolean workflowAlreadyExist() throws DAOException {
+    private boolean workflowAlreadyExist() throws DAOException, IOException, WorkflowException {
         List<Task> processTasks = aggregatedProcesses.get(currentTasks).get(0).getTasks();
-        List<Template> allTemplates = ServiceManager.getTemplateService().getAll();
-        for (Template template : allTemplates) {
+        System.out.println("processTasks: " + processTasks.size());
+        List<Workflow> allWorkflows = ServiceManager.getWorkflowService().getAll();
```

Comment:
You probably don't need to save the workflows to a separate varaible anymore when you remove the `System.out.println("alWorkfows: " + allWorkflows.size() );`.